source/common/config/utility.cc

```diff
@@ -113,40 +113,43 @@ void Utility::checkApiConfigSourceNames(
 
   if (is_grpc) {
     if (!api_config_source.cluster_names().empty()) {
-      throw EnvoyException(fmt::format("envoy::api::v2::core::ConfigSource::(DELTA_)GRPC "
+      throw EnvoyException(fmt::format("{}::(DELTA_)GRPC "
```

Comment:
nit: can we re-wrap these format strings now that they're shorter

Reply:
sure. Done. PTAL